patterns/1-initial/migrating-from-innersource-to-open-source.md

```diff
@@ -0,0 +1,125 @@
+## Title
```

Comment:
General question for this new pattern:

1. Are you aware of public references that describe how to open source a project? We could add these here.
2. Are we documenting an open source pattern here, or an InnerSource pattern? I suspect there are benefits of this in the InnerSource context. Maybe thinking about this questions helps us to decide which elements of this new pattern we want to emphasize.

Reply:
Added references

It inherently overlaps, but certainly leans into InnerSource.